scripts/localk8s/README.md

```diff
@@ -0,0 +1,52 @@
+# Deployment to local Kubernetes
+
+The scripts in the `scripts/localk8s` directory can be used to deploy the refarch-kc microservices and backing services to a local Kubernetes cluster.
+
+## Installation
+
+The entire installation process can be run by executing the `install.sh` or `install.bat` script. This script will invoke:
+- `install-infra.sh` to set up the Kafka and Postgres backing services,
+- `install-app.sh` to configure the configmaps, Kafka topics and secrets, and deploy the microservices,
+- `run-integration-tests.sh` to execute the integration tests against the deployment as a Kubernetes job.
+
+If desired, these steps can each be run individually by running the appropriate script.
+
+### Namespaces
+
+During the installation, a set of namespaces will be created to hold the components:
+- `strimzi` to hold the Strimzi operator,
+- `kafka` to hold the Kafka cluster that Strimzi will create for us,
+- `postgres` to hold the Postgres operator and DB instance,
+- `shipping` to hold the application microservices and configuration.
+
+### Infrastructure
+
+The infrastructure set up by `install-infra.sh` consists of:
+- The Strimzi Kafka Operator, installed using Helm, configured to watch the `kafka` namespace,
+- A Kafka cluster named `my-cluster`, with a single replica, ephemeral storage, and no TLS,
+- A ServiceAccount called `pgserviceaccount`,
+- A Postgres DB installed using the Bitnami Postgresql Helm chart, configured with `pgserviceaccount`.
+
+### Application and configuration
+
+The configuration and services deployed by `install-app.sh` are all contained within the `shipping` namespace. The configuration consists of:
+- A ServiceAccount called `kcserviceaccount`,
+- Secrets for Postgresql to allow connection to the DB created in the previous step,
+- ConfigMap `kafka-brokers` with the address of the Kafka cluster created in the previous step,
+- ConfigMap `kafka-topics` with the names of the Kafka topics we'd like to use,
+- A set of Strimzi KafkaTopic CRs to create the Kafka topics in our cluster,
+
+Each of the microservices is then installed using its respective Helm chart. The repos will be cloned using the existing `clone.sh` script if they do not already exist at the same directory level as the `refarch-kc` repo.
+
+### Integration tests
+
+Running the `run-integration-tests.sh` script will create a Kubernetes job using the `itg-tests/es-it/ReggerItgTests.yaml` definition, with the `KAFKA_ENV` variable set to `LOCAL`, and the topic names set to their default values (without the `itg-` prefix).
```

Comment:
There is a (probably autocorrect?) typo in the reference to `itg-tests/es-it/ReggerItgTests.yaml`. It should be `itg-tests/es-it/ReeferItgTests.yaml` instead.

Reply:
Actually that section is out of date, as I'm now creating `itg-`-prefixed topics and temporarily changing the configuration to use them. I'll update it.